docs/documentation/server_admin/topics/clients/proc-evaluating-client-scopes.adoc

```diff
@@ -15,7 +15,7 @@ image:images/client-scopes-evaluate.png[]
 
 [NOTE]
 ====
-To send a custom value for a *scope* parameter from your application, see the link:{adapterguide_link_latest}#_javascript_adapter[javascript adapter section], for javascript adapters.
+To send a custom value for a *scope* parameter from your application, see the *Keycloak JavaScript adapter* in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section, for javascript adapters.
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).